src/vs/code/electron-main/app.ts

```diff
@@ -832,7 +839,8 @@ export class CodeApplication extends Disposable {
 				urisToOpen: macOpenFiles.map(file => this.getWindowOpenableFromPathSync(file)),
 				noRecentEntry,
 				waitMarkerFileURI,
-				initialStartup: true
+				initialStartup: true,
+				remoteAuthority: undefined /* always local */
```

Comment:
Is that needed here though? Shouldn't the fact that we have `urisToOpen` be enough to decide on the `remoteAuthority`?